app/src/main/java/com/example/mvpmusicapp/data/source/MusicDataSource.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.mvpmusicapp.data.source
+
+import com.example.mvpmusicapp.data.source.local.task.OnResultCallback
+
+interface MusicDataSource {
```

Comment:
```suggestion
interface MusicsDataSource {
```
thường thì cái DataSource sẽ là số nhiều